### PR TITLE
Full Russian Translation [Updated]

### PR DESCRIPTION
**Pull request by aayojezuzchrezt**
Updated locales (synced and translated them after last update). (Not sure if I get "Good rankings", so left original next to translated, might fix it later)

### CODE REVIEW

File: website/public/locales/ru/side_menu.json
Comment (Klotske) [inline website/public/locales/ru/side_menu.json:12]: Small typo :)
Should be "Доска"

**Comment by AbdBarho** [comment]
Blocked by #1327

**Review by Klotske — APPROVED** [review]
LGTM

**Comment by aayojezuzchrezt** [comment]
Done, I guess

**Review by fozziethebeat — CHANGES_REQUESTED** [review]
Will approve after the new merge conflicts are resolved

**Review by AbdBarho — APPROVED** [review]

**Review by fozziethebeat — APPROVED** [review]

**Pull request merged**
